docs/blog/2021-01-25-zmk-sotf-4.md

```diff
@@ -0,0 +1,190 @@
+
+#### Central/Peripheral Selection
+
+TODO: Spell this out.
+
+https://github.com/zmkfirmware/zmk/pull/510 [petejohanson]
+
+#### Improved Docker Containers
+
+As part of the Zephyr 2.4. prep work, [innovaker], along with lots of testing and input from [mcrosson], developed a brand new pair of [Docker images](https://github.com/zmkfirmware/zmk-docker) which is now published to Docker Hub as [zmkfirmware/zmk-build-arm](https://hub.docker.com/repository/docker/zmkfirmware/zmk-build-arm) and [zmkfirmware/zmk-dev-arm](https://hub.docker.com/repository/docker/zmkfirmware/zmk-build-arm).
+
+The previously blogged VSCode + Docker integration, as well as our GH Action build automation was all moved over to the new images.
+
+#### Settings Debounce
+
+[nicell] contributed settings debounce work, to help avoid unnecessary extra writes to flash when making various changes that should be saved,
+such as the active BLE profile, external VCC on/off, etc.
+
+## New Shields
+
+- Jorne & Jian in [#331](https://github.com/zmkfirmware/zmk/pull/331) - [krikun98](https://github.com/krikun98)
+- tidbit in [#424](https://github.com/zmkfirmware/zmk/pull/424) - [mcrosson](https://github.com/mcrosson)
+- Helix in [#429](https://github.com/zmkfirmware/zmk/pull/429) - [KingCoinless](https://github.com/KingCoinless)
+- BFO-9000 in [#472](https://github.com/zmkfirmware/zmk/pull/472) - [pbz](https://github.com/pbz)
+- CRBN in [#493](https://github.com/zmkfirmware/zmk/pull/483) - [ReFil](https://github.com/ReFil)
+- Eek in [#529](https://github.com/zmkfirmware/zmk/pull/529) - [MangoIV](https://github.com/MangoIV)
+
+## New Boards
+
+- BDN9 Rev2 in [#557](https://github.com/zmkfirmware/zmk/pull/557) - [petejohanson]
+
+## Sponsorship
+
+Since it's inception, quite a few users have inquired whether they could sponsor any of the contributors involved in ZMK. Although we are not intending to directly fund any individual contributors for their work on ZMK, there _is_ good that can come from folks sponsoring ZMK.
+
+You can see the full discussion on [#497](https://github.com/zmkfirmware/zmk/issues/497), but some items that are being considered with sponsorship funds:
+
+- Hiring a designer to complete the logo/mascot work.
+- Creating stickers to send as thank-yous to first time contributors.
+- Hosting costs for GitHub Pro.
+- Other hosting costs, e.g. Docker Hub.
+
+For anyone looking to contribute, you can find the [ZMK Firmware project](https://opencollective.com/zmkfirmware) is now set up on [Open Collective](https://opencollective.com).
+
+## Coming Soon!
+
+Some items listed in the last coming soon section are still under active development.
+
+- A power profiler page for the website, to help users estimate their battery life for a given keyboard - [Nicell]
+- A keymap converter to automatically update keymaps to the new codes and use of `&kp` everywhere - [joelspadin]
```

Comment:
This one was merged a while ago! https://zmkfirmware.dev/docs/codes/keymap-upgrader

Reply:
Yup. Added as a section to the keymap part of the post, with link to the upgrader for folks to use.